src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java

```diff
@@ -51,7 +51,7 @@ public class StudentExamService {
 
     private static final String EXAM_EXERCISE_START_STATUS_TOPIC = "/topic/exams/%s/exercise-start-status";
 
-    private final Logger log = LoggerFactory.getLogger(StudentExamService.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(StudentExamService.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(StudentExamService.class);
```
`log` is used consistently in Artemis for static and non-static logger attributes.

Reply:
I'm reverting the change, but somebody should update the teamscale checks as this is a reason for multiple findings at the moment.